examples/text_labels_and_annotations/mathtext_fontfamily_example.py

```diff
@@ -0,0 +1,36 @@
+"""
+========================
+Math Fontfamily Example
+========================
```

Comment:
```suggestion
===============
Math fontfamily
===============
```

We don't use example/demo anymore in the title, because that's unnecessary clutter.